cmd/vulcan-certinfo/main.go

```diff
@@ -216,6 +218,9 @@ func main() {
 		}
 
 		for _, certInfo := range checker.certificateInfo {
+			certInfo.AffectedResource = checker.fingerprint
+			certInfo.AffectedResourceString = target
```

Comment:
How would you see using the `opt.Port` instead?

Reply:
Thanks for the review @julianvilas 
Agree, as we've discussed, it is convenient to add `opt.Port` in the AffectedResource and AffectedResourceString attributes.
Done here [720fa7b](https://github.com/adevinta/vulcan-checks/pull/150/commits/720fa7b458e09317e79bde705c0dbc4e5f722ed2)

Reply:
Thx!